Architecture.md

```diff
@@ -60,7 +60,13 @@ It is used because it discovers when containers are started/stopped and can dyna
 
 #### Database
 
-**_TODO: [CLIENT] fill for the unified database schema_**
+The requirement for an ACID (Atomicity, Consistency, Isolation, and Durability) database led us to choose a relational database. For our application there wasn't a need to have a multitude of adaptable fields and the benefits of having consistent and structured data with relationships between different objects were more important factors.
```

Comment:
I think you should emphasize more on transactions, and how important they are to our use case.

Reply:
Will do